crates/driver/src/extract_defrefs.rs

```diff
@@ -194,24 +196,30 @@ async fn async_extract_def_refs(
 async fn merge_defrefs(
     concurrent_io_operations: &Semaphore,
     path_sha_to_merged_defrefs: &'static Path,
-    directory: String,
+    entry: String,
     project_conf: &'static ProjectConf,
     mut work_items: Vec<ProcessedFile>,
     sha_of_conf_config: Arc<String>,
+    no_aggregate_source: bool,
 ) -> Result<(String, ExtractedMapping)> {
     work_items.sort_by(|a, b| a.sha256.cmp(&b.sha256));
 
     let merged_sha = Sha256Value::hash_iter_bytes(
         work_items
             .iter()
             .map(|e| e.sha256.as_bytes())
-            .chain(std::iter::once(sha_of_conf_config.as_bytes())),
+            .chain(std::iter::once(sha_of_conf_config.as_bytes()))
+            .chain(std::iter::once(if no_aggregate_source {
+                &b"1"[0..1]
```

Comment:
nit: this looks like we are using ascii `"1"` for true, rather than the byte with value 1. Am I understanding that correctly? I'd personally rather use byte == 0 and byte == 1 not the ascii.

Reply:
Yea, you're probably right.

Reply:
Addressed in https://github.com/bazeltools/bzl-gen-build/pull/117/commits/312fd34366f893136280e71bfe20d30c2b4f87f2